knife.txt

```diff
@@ -1 +1,3 @@
-So what the pen is mightier than I. I'll fucking open you up. Make you bleed.
```

Comment:
Why did you remove this line then added it back in.

Reply:
I don't answer to you! Errr..I sort of do.